jenkins-x.yml

```diff
@@ -6,48 +6,78 @@ pipelineConfig:
       pipeline:
         agent:
           image: gcr.io/jenkinsxio/builder-terraform:2.1.142-761
+        env:
+          - name: VERIFY_KEY_VAULT_IMAGE_NAME
+            value: gcr.io/jenkinsxio/azure_verify_key_vault:1.0
```

Comment:
this is the image that will be produced right?  I think if we use
```
gcr.io/jenkinsxio/azure_verify_key_vault:$(inputs.params.version)
```
then a snapshot image will be build and tagged instead which helps during CI builds.  Each CI build will then have a unique image, then once PR is merged a proper released tag would be used.

Reply:
Yep, all good. Suspected we could do this  looking at other pipelines but I didn't have time to fully understand what was going on so just hard coded. I'll push a modified version as you suggest soon